pages-config.json

```diff
@@ -14,6 +14,7 @@
   "assetRoot":        "/guides-template",
   "fileLockWaitTime": 30000,
   "fileLockPollTime": 1000,
+  "secretKeyFile":    "/home/ubuntu/.18f-pages.secret",
```

Comment:
This is actually here to generate an initial template via `bin/18f-pages print-template`. The "real" config is in 18F/pages.